src/measure_theory/measurable_space_def.lean

```diff
@@ -134,6 +134,10 @@ lemma measurable_set.Inter [encodable β] {f : β → set α} (h : ∀ b, measur
 measurable_set.compl_iff.1 $
 by { rw compl_Inter, exact measurable_set.Union (λ b, (h b).compl) }
 
+lemma measurable_set.cond {A B : set α} (hA : measurable_set A) (hB : measurable_set B)
```

Comment:
Could you put that lemma in measurable_space instead of measurable_space_def? Or is it needed in a file that does not import measurable_space?

If you can put it in measurable_space, it would be nice to add the `[measurability]` attribute, in order to make it usable by the measurability tactic. If you cannot or if you think that it makes more sense to keep it in measurable_space_def, please add the name of that lemma to the list in the file measure_theory/tactic (line 64).

Reply:
I've moved it too the end of `measurable_space` now since I don't really know where to put it. I hope thats okay